common/helpers/move/can-cancel-move.js

```diff
@@ -0,0 +1,37 @@
+function checkCanCancelMove(canAccess, status, allocation) {
+  if (canAccess('move:cancel')) {
+    if (['requested', 'booked'].includes(status)) {
+      if (!allocation) {
+        return true
+      }
+    }
+  }
+
+  return false
+}
+
+function checkCanCancelMoveProposed(canAccess, status) {
+  if (canAccess('move:cancel:proposed')) {
+    if (status === 'proposed') {
+      return true
+    }
+  }
+
+  return false
+}
```

Comment:
```suggestion
function checkCanCancelMoveProposed(canAccess, status) {
  return canAccess('move:cancel:proposed') && status === 'proposed'
}
```

I realise you only renamed the file, but this feels like a nice refactor.

Reply:
I agree. Initially I was trying to keep changes down to hopefully make it easier to review but I agree with this refactor :) I'm also more a fan of combined return statements over nested `if`s.

Will also add this in when updating the tests.